python/llm/example/GPU/PyTorch-Models/Model/openai-whisper/README.md

```diff
@@ -0,0 +1,141 @@
+# Whisper
+In this directory, you will find examples on how you could apply IPEX-LLM INT4 optimizations on general pytorch models, for example Openai Whisper models. For illustration purposes, we utilize the [whisper-tiny](https://github.com/openai/whisper/blob/main/model-card.md) as a reference Whisper model.
```

Comment:
In this directory, you will find examples of how to use IPEX-LLM to optimize OpenAI Whisper models within the `openai-whisper` Python library.